rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -489,3 +491,25 @@ def test_eth_deposit(
             extra_data=None,
         ),
     ]
+
+
+def test_maybe_reshuffle_events():
+    """
+    Tests that `maybe_reshuffle_events` works correctly.
+    Especially tests that there are no duplicated indices produced.
+    """
+    event_a = make_ethereum_event(1)
+    event_b = make_ethereum_event(2)
+    event_c = make_ethereum_event(3)
+
+    events = [event_a, event_b, event_c]
+    assert events == [event_a, event_b, event_c]  # sanity check
```

Comment:
You assigned the variable above yourself.

The `sequence_index` check is acceptable but this one seems like a tautology.